src/datasets/demo_only.py

```diff
@@ -21,11 +21,11 @@ def create_demo_data(env: BaseEnv, train_tasks: List[Task]) -> Dataset:
             task,
             env.simulate,
             env.predicates,
-            CFG.max_num_steps_check_policy,
-            annotate_traj_with_goal=True)
+            CFG.max_num_steps_check_policy)
+        assert solved, "Oracle failed on training task."
+        traj = LowLevelTrajectory(traj.states, traj.actions, task.goal)
```

Comment:
add goal to traj, this is temporary since we're gonna change all this soon anyway